src/setup.rs

```diff
@@ -21,11 +18,17 @@ use log::info;
 use std::str::FromStr;
 
 pub(crate) fn initial_state(config: config::SpotifydConfig) -> main_loop::MainLoop {
-    let mut mixer = {
+    let mixer_config = MixerConfig {
+        device: config.device_name.clone(),
+        volume_ctrl: librespot_playback::config::VolumeCtrl::default(),
+        control: "".to_owned(),
+        ..Default::default()
+    };
```

Comment:
I'm a bit confused by this. As far as I can tell, this is only used by the `SoftVol` mixer below. The code for this mixer actually only cares about the `volume_ctrl` field. And since you're just setting that to default here, wouldn't it be easier to just do `MixerConfig::default()` below?

Reply:
Good point. Though I wanted to make sure the device name was right and to be extra explicit, but using the default looks nicer.